README.md

```diff
@@ -48,10 +48,28 @@ The returned data is the posted JSON payload.
 With the above curl example it would be `OK`. 
 The log file will thus show `Webhook called with data: OK`.
 
-See example pipelines in the `examples` directory.
+For illustration, see the [scripted pipeline example](examples/scripted_pipeline).
```

Comment:
it would be great to use the example to run the JenkinsRule test for the next time. So you know that it's always up to date.

Reply:
Good idea. Thanks for the suggestion.